plutus-chain-index/app/Marconi/Orphans.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Marconi.Orphans where
+
+import Cardano.Api (BlockHeader, BlockNo (BlockNo), ChainPoint (..), ChainTip (ChainTip, ChainTipAtGenesis), Hash,
+                    SlotNo (SlotNo), serialiseToRawBytesHexText)
+import Prettyprinter (Pretty (pretty), (<+>))
+
+instance Pretty ChainTip where
+  pretty ChainTipAtGenesis   = "genesis"
+  pretty (ChainTip sn ha bn) = "slotNo" <+> pretty sn <+> "hash" <+> pretty ha <+> "blockNo" <+> pretty bn
```

Comment:
```suggestion
  pretty (ChainTip sn ha bn) = "ChainTip(SlotNo <+> pretty sn <+> ", BlockHash" <+> pretty ha <+> ", BlockNo" <+> pretty bn
```

Reply:
Isn't the point of the `Pretty` class to format the data in nice human readable form? How would the user know which is the slot number and which is the block number?